guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -226,7 +226,7 @@ endif::[]
 ifdef::satellite[]
 . Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
 endif::[]
-. If you require SELinux to be in enforcing mode, run the following command before changing SELinux to enforcing mode:
+. For {Project} only and not {SmartProxy}, if you require SELinux to be in enforcing mode, run the following command before changing SELinux to enforcing mode:
```

Comment:
This is the only step that does not need to be executed on a SmartProxy (and in the Satellite case won't even work, as we do not ship the mentioned packages in the Capsule repository)